components/LeagueEntries/LeagueEntries.test.tsx

```diff
@@ -4,14 +4,16 @@ import React from 'react';
 
 describe('LeagueEntries', () => {
   it(`renders 'default' state without a pick made`, () => {
-    render(<LeagueEntries entryName="Entry 1" linkUrl='' />);
+    render(<LeagueEntries entryName="Entry 1" linkUrl="" />);
```

Comment:
I don't see a test that checked the linkUrl. It's a required prop so it should be prioritized to be tested.